### PR TITLE
[90] All packages are lowercase

### PR DESCRIPTION
**Pull request by sam-i-am012**
Forgot to make a PR for this.

### CODE REVIEW

File: src/test/java/org/irods/operations/ResourceOperationsTest.java [hunk 1]
```diff
@@ -3,9 +3,9 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.irods.IrodsHttpClient;
-import org.irods.properties.DataObject.DataObjectRemoveParams;
-import org.irods.properties.DataObject.DataObjectWriteParams;
-import org.irods.properties.Resource.ResourceCreateParams;
+import org.irods.properties.dataObject.DataObjectRemoveParams;
```
Comment (korydraughn) [inline src/test/java/org/irods/operations/ResourceOperationsTest.java:6]: Is `dataObject` a package or class?

Comment (sam-i-am012) [reply src/test/java/org/irods/operations/ResourceOperationsTest.java:6]: It's a package that has the params classes.

Comment (korydraughn) [reply src/test/java/org/irods/operations/ResourceOperationsTest.java:6]: Ok. According to https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html, `dataObject` should be `data_object`.

Let's do that for all package names which use two or more words.

Comment (korydraughn) [reply src/test/java/org/irods/operations/ResourceOperationsTest.java:6]: That is, use underscores to separate multiple words in package name elements.

Comment (sam-i-am012) [reply src/test/java/org/irods/operations/ResourceOperationsTest.java]: Ohh, I understand, makes sense. Done.

**Comment by sam-i-am012** [comment]
All existing review comments have been resolved.

**Comment by trel** [comment]
Typo in your last commit message… separate

**Comment by sam-i-am012** [comment]
Oops, there we go.

**Review by korydraughn — COMMENTED** [review]
Squash it to one commit and this will be ready.

**Comment by trel** [comment]
and there are now conflicts...

**Comment by sam-i-am012** [comment]
Conflicts resolved and squashed to one commit, all tests pass.

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by sam-i-am012** [comment]
Done!

**Pull request merged**